05-UI-ReactJS/topic-05/src/App.css

```diff
@@ -0,0 +1,90 @@
+.Container {
+  max-width: 250px;
+  width: 100%;
+  margin: auto;;
+}
+
+.MovieInput {
+  float: left;
+}
+
+.MovieTitleUI, .MovieYearUI, .MovieDurationUI, .ActorNameUI, .ActorYearUI {
+  float: left;
+  margin-bottom: 5px;
+  width: 100%;
+}
+
+.ButtonSubmit {
+  margin: 5px 0px 5px;
+  float: right;
+}
+
+.ActorSubmit {
+  float: left;
+  margin: 5px 0px 5px;
+}
+
+.ActorList {
```

Comment:
It can be spelled like that also! Is british english i think!

http://www.wordreference.com/es/translation.asp?tranword=favourite